### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by mikheillomidzeq**
There was some manual changes in documentation. This PR moves all manual changes into templates and examples. Now we are getting same documentation via command `make generate-help` .

### CODE REVIEW

File: templates/guides/getting-started.md.tmpl [hunk 1]
```diff
@@ -80,4 +81,31 @@ output "url" {
...
\ No newline at end of file
+```
+
+
+### Available cloud providers and regions
+```yaml
+aws:
+- us-east-1
+- us-west-1
+- eu-central_lega-y
```
Comment (Robert-Stam) [inline templates/guides/getting-started.md.tmpl:92]: is this correct, should this one be deleted here?

**Review by Robert-Stam — APPROVED** [review]
1 question, further LGTM

File: templates/guides/getting-started.md.tmpl (hunk 1 above)
Comment (mikheillomidzeq) [reply templates/guides/getting-started.md.tmpl:92]: No, it is a mistake. 

Thank you for noticing.

**Review by Robert-Stam — APPROVED** [review]
LGTM, well done!!

**Pull request merged**